### PR TITLE
feat: add prettier configs

### PR DESCRIPTION
**Pull request by jquense**
I think this is the best way to do this...the prettier rules really need to come at the end.

so like `extends: ['eslint-config-4catalyzer/prettier']`

### CODE REVIEW

File: packages/eslint-config-4catalyzer/rules.js [hunk 1]
```diff
@@ -49,4 +49,5 @@ module.exports = {
   // We very seldom intentionally use async functions without await.
   'require-await': 'error',
   'import/prefer-default-export': 'off',
+  'lines-between-class-members': 'off',
```
Comment (taion) [inline packages/eslint-config-4catalyzer/rules.js]: i like this though ):

Comment (taion) [reply packages/eslint-config-4catalyzer/rules.js]: oh does `--fix` not handle that?

Comment (jquense) [reply packages/eslint-config-4catalyzer/rules.js]: maybe it does...the docs say it does, but i had to do most of them by hand for react-bootstrap

Comment (taion) [reply packages/eslint-config-4catalyzer/rules.js]: \\: i like this rule though

Comment (jquense) [reply packages/eslint-config-4catalyzer/rules.js]: fiiiiiine

**Comment by taion** [comment]
but my question is more... in practice, we always have to override our eslintrc files anyway to add the prettier/prettier rule, no?

so given that we need to add some overrides there anyway, does it save that much to pull in the config in the base?

**Comment by jquense** [comment]
Yeah idk, we could also add the rule here, i'm just less convinced that linting via the prettier rule is a Good Idea, tho we are doing it everywhere. We can also handle this via scaffolding as well.

**Comment by taion** [comment]
I feel like just pulling in the config in projects is the same amount of complexity. Part of the reason I set things up with the multiple packages here is that various properties interact in non-obvious ways, e.g. disable the prop type rule when using Flow and React...

**Pull request closed without merging**